frontend/src/providers/QueryBuilder.tsx

```diff
@@ -916,6 +916,12 @@ export function QueryBuilderProvider({
 		],
 	);
 
+	useEffect(() => {
+		if (location.pathname === ROUTES.INFRASTRUCTURE_MONITORING_HOSTS) {
+			setCurrentQuery(initialQueriesMap.metrics);
```

Comment:
what why ? check how it is done for logs / traces explorer.

Reply:
@vikrantgupta25  Removed this code and updated it according to the traces and logs explorer.